src/components/ServiceRequestForm.jsx

```diff
@@ -310,7 +314,7 @@ const ServiceRequestForm = (props, errors, touched) => {
 					const value = requestType.toLowerCase();
 					const subCategories = getSubCategories(Categories, value ? value : value);
 					setSubCategories(subCategories);
-					if (localProps.values['petType'] !== '') {
+					if (petType !== '') {
```

Comment:
this can probably be `if (petType) {`

Reply:
yup and the above of requestType =='' has been removed as well.